app/controllers/welcome_controller.rb

```diff
@@ -2,63 +2,16 @@ class WelcomeController < ApplicationController
   skip_before_action :verify_authenticity_token, only: [:show]
```

Comment:
After this PR has been merged and deployed, we can update finder-frontend to POST to the registration path, at which point this can be removed and `show` becomes a GET route which just redirects the user based on the presence of a session.

Reply:
Or not.  Keeping the cross-site stuff in this controller has advantages.